assets/javascripts/modules/analytics/ga.js

```diff
@@ -64,8 +64,19 @@ define(['modules/analytics/analyticsEnabled',
         ga('jellyfishGA.send', 'pageview');
     }
 
+    function trackEvent(eventLabel) {
+        var upgrading = (eventLabel === 'Rate Plan Change' && guardian.pageInfo.productData.initialProduct !== guardian.pageInfo.productData.productPurchasing) ? 1 : 0;
+        ga('membershipPropertyTracker.send', 'event', {
+            eventCategory: 'Subscriptions Checkout',
+            eventAction:  guardian.pageInfo.productData.productType,
+            eventLabel: eventLabel,
+            dimension11: guardian.pageInfo.productData.productType + ' - ' + guardian.pageInfo.productData.productPurchasing,
```

Comment:
dimension11 and metric1 are not very reader friendly, is there any way to be more sure by inspection away that those are correct?